README.md

```diff
@@ -31,10 +31,40 @@ Clone each of the repositories and run `make push-image-devel`
  - [Alloy](https://github.com/metal-toolbox/alloy/)
  - [Flasher](https://github.com/metal-toolbox/flasher/)
  - [Flipflop](https://github.com/metal-toolbox/flipflop)
- - [Fleetscheduler](https://github.com/metal-toolbox/fleet-scheduler) (OPTIONAL, enable in values.yaml fleetscheduler.enable)
 
 This will build and push the container images to the local container registry.
 
+### 1.1. Point to local repositories (Optional)
+
+Some of our services have moved to be self contained helm charts. They are included within Chart.yaml as dependencies.
+
+ - [Fleetscheduler](https://github.com/metal-toolbox/fleet-scheduler) (OPTIONAL, enable in values.yaml fleetscheduler.enable)
+
+To replace a dependency with a local version of the dependency you are working on, you can edit the dependency within the Chart.yaml, and can refer to the [helm docs](https://helm.sh/docs/helm/helm_dependency/) for more information.
+
+For example: To replace fleet-scheduler's upstream helm chart with a local custom version, you must do the following:
+- Clone [Fleet-Scheduler](https://github.com/metal-toolbox/fleet-scheduler). Preferably in the same directory as the [Sandbox](https://github.com/metal-toolbox/sandbox)
```

Comment:
So `fleet-scheduler` and `sandbox` need to be siblings, not parent-child, yes? Is this just an exercise in finding the file correctly via the URI?

Reply:
Correct. Both the sandbox and fleet-scheduler will need to have the same parent directory. Maybe i should make that a bit more clear?

Reply:
I mean, I had to read it twice, but I was skimming it and the example didn't match my mental model. Going back it was clearer. You might consider explicitly stating the directories should be siblings for the purposes of being opinionated in your example.